zkp/golang/lib/smt/node.go

```diff
@@ -189,14 +207,25 @@ func (idx *nodeIndex) Equal(other NodeIndex) bool {
 	return idx.BigInt().Cmp(other.BigInt()) == 0
 }
 
-func (idx *nodeIndex) IsBitPositionOn(pos uint) bool {
-	if pos < 0 || pos >= 256 {
+// getPath returns the binary path, from the root to the leaf.
+func (idx *nodeIndex) ToPath(levels int) []bool {
```

Comment:
Would it make sense to define a type for "path" instead of just using bool? I see that paths use a convention of "left = false, right = true", so we could consider using constants for left/right.

Reply:
good idea. if it's OK with you, will add in a separate PR to avoid merge pain with the subsequent PRs which had significant code structure updates